modules/dr-recover-lost-control-plane-hosts.adoc

```diff
@@ -11,17 +11,15 @@ Follow these steps to recover from the loss of the majority of master hosts, lea
 
 * Access to the cluster as a user with the `cluster-admin` role.
 * SSH access to a remaining master host.
-* An etcd backup file that includes an etcd snapshot and static Kubernetes API server resources.
-+
-This backup file must be in the format of `snapshot_db_kuberesources_<datetimestamp>.tar.gz`.
+* A backup directory containing both the etcd snapshot and static Kubernetes API server resources. The file names in the directory must be in the following formats: `snapshot_<datetimestamp>.db` and `static_kuberesources_<datetimestamp>.tar.gz`.
```

Comment:
while it is kind of implicit: both files must match, i.e. cannot be mixed for different backups.

Reply:
Updating to add "taken from the same backup" if that works.